jipdate.py

```diff
@@ -267,16 +267,23 @@ def parse_status_file(jira, filename):
                 (i,c) = issue_comments[-1]
                 issue_comments[-1] = (i, c + line)
 
+    issue_upload = []
     print("These JIRA cards will be updated as follows:\n")
     for (idx,t) in enumerate(issue_comments):
         (issue,comment) = issue_comments[idx]
 
         # Strip beginning  and trailing blank lines
         comment = comment.strip()
-        issue_comments[idx] = (issue, comment)
+
+        if comment == "":
+            print("[%s] is empty and has been deleted" % (issue))
```

Comment:
The issue is deleted from the list of issues to be updated :-)
jiar api will return an error if the comment is empty